prover/crypto/keccak/subtle/xor.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2022 The Go Authors. All rights reserved.
```

Comment:
Likewise, we should remove the header or avoid these optimization if we copy non-apache code.